public/index.php

```diff
@@ -29,12 +29,230 @@
+      <br>Update teams for your lectures
+    </li><br>
+    <li>
+      <h4>Account settings</h4>
+      Change your account settings
+    </li><br>
+    <li>
+      <h4>emails</h4>
+      View your internal emails 
+      <br>Send emails internally to other users
+    </li><br>
+    <li>
+      <h4>Meetings</h4>
+      View all meetings created by students in your lectures
+    </li><br>
+    <li>
+      <h4> Courses, Lectures and Sections</h4>
+      View the lecture pages for your assigned lectures
+    </li><br>
+    <li>
+      <h4>Students</h4>
+      Upload student lists to specific sections through the course lecture pages of the lectures you are assigned to
+    </li><br>
+    <li>
+      <h4>Marked Entities</h4>
+      View all marked entities for your lectures
+      <br>Create marked entities for your lectures
+      <br>Update marked entities for your lectures
+      <br>Comment on students uploaded files for your lectures' marked entities
+      <br>Create Discussions related to marked entities visible only to TAs and instructors  
+      <br>Reply to messages on marked entity Discussions
+      <br>Create polls in marked entiy discussion messages
+      <br>Comment on marked entity discussion messages from students
+    </li><br>
+    <li>
+      <h4>Discussions</h4>
+      View lecture discussion pages for your lectures
+      <br>Create messages in your lecture discussion pages
+      <br>Reply to messages in your lecture discussion pages
+      <br>Create polls in your lecture discussion pages 
+      <br>Comment on messages in your lecture discussion pages
+    </li>
+    <li>
+      <h4>TAs</h4>
+      Create TAs and automatically assign them to a course sections associated to your lectures
+      <br> Assign existing TAs to a course section associated to your lectures that has no TA
+    </li>
+  </ul>
+  <?php } elseif ($user_role == "ta") { ?>
+    <h3>As a TA you may: </h3>
+
+<li>
+    <h4>Announcements</h4>
+    View all system wide announcements
+  </li><br>
+  <li>
+    <h4>Teams</h4>
+    View teams for your lectures associated to your assigned sections
+  </li><br>
+  <li>
+    <h4>Account settings</h4>
+    Change your account settings
+  </li><br>
+  <li>
+    <h4>emails</h4>
+    View your internal emails 
+    <br>Send emails internally to other users
+  </li><br>
+  <li>
+    <h4>Meetings</h4>
+    View all meetings created by students in lectures associated with your sections
+  </li><br>
+  <li>
+    <h4> Courses, Lectures and Sections</h4>
+    View the lecture pages for lectures associated to your assigned sections
+  </li><br>
+  <li>
+    <h4>Students</h4>
+    View all students registered in lectures associated to your course sections
+  </li><br>
+  <li>
+    <h4>Marked Entities</h4>
+    View all marked entities for your lectures
+    <br>Comment on students uploaded files for your lectures' marked entities
+    <br>Create Discussions related to marked entities visible only to TAs and instructors  
```

Comment:
here too

Reply:
I checked though and on marked entities when I create a discusssion as a TA that discussion isnot visible to students